test/integration/scenario.ts

```diff
@@ -94,10 +85,8 @@ describe('Scenario test multi-pair', function () {
           const { address } = await getPlaceOrderData(orderReceipt)
 
           order = await ethers.getContractAt('Order', address)
-          expect(isClose(await tokenFrom.balanceOf(address), value)).to.be.true
-          expect(
-            isClose(await tokenFrom.balanceOf(stonks.getAddress()), BigInt(0))
-          ).to.be.true
+          expect(isClose(await tokenFrom.balanceOf(address), value, 2n)).to.be.true
+          expect(isClose(await tokenFrom.balanceOf(stonks.getAddress()), BigInt(0), 2n)).to.be.true
```

Comment:
The `.getAddress()` call is redundant. Ethers accepts the contract instances as part of the `AddressLike` type.